portal/src/main/webapp/js/src/patient-view/util/ClinicalAttributesUtil.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+/**
+ * Functions for dealing with clinical attributes.
+ */
+window.ClinicalAttributesUtil = (function() {
+  /**
+   * Clean clinical attributes. Useful for rounding numbers, or other types of
+   * data cleanup steps. Probably differs per institution.
+   * @param {object} clinicalData - key/value pairs of clinical data
+   */
+  function clean(clinicalData) {
+    // Shallow Copy clinicalData
+    var cleanClinicalData = $.extend({}, clinicalData);
+    var NULL_VALUES = [
```

Comment:
I think there’s no need to add a third place where these values are listed, and can potentially get out of synch with the documentation (it already introduces one here: `[unkown]`). @pieterlukasse told me that the loading code now translates all of them to the same database representation, and even for previously-loaded databases that do already contain these values it would be cleaner to translate them in the Java layer using the same enum before sending them to the client.